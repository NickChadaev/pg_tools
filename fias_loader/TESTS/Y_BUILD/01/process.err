ЗАМЕЧАНИЕ:  отношение "__adr_area_fias" уже существует, пропускается
ЗАМЕЧАНИЕ:  отношение "__adr_area_fias" уже существует, пропускается
