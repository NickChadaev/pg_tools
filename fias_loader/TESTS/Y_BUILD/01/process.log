2023-11-02 14:55:11 ./
127.0.0.1 5434 unsi_test_01 postgres stage_gar_c_01.csv 2023-10-26
--------------------------------------------------------
[14:55:11] ... Start process
[14:55:11] ...  -- Version
[14:55:11] ...  -- Очистка от данных
[14:55:13] ...  -- Set UNLOGGED
[14:55:16] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/AS_ADDHOUSE_TYPES_20231026_1b8ca463-4b89-4a94-96e9-2567cd8bae55.XML
[14:55:16] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/AS_ADDR_OBJ_TYPES_20231026_f94d64d9-84f7-4717-8c8c-38f9f18a8f85.XML
[14:55:16] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/AS_APARTMENT_TYPES_20231026_b7af29ce-b522-4733-8996-50a115c9ee6c.XML
[14:55:16] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/AS_HOUSE_TYPES_20231026_766c4e6c-f5bf-416c-b939-76e5d7169016.XML
[14:55:16] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/AS_NORMATIVE_DOCS_KINDS_20231026_247eb17b-646d-4dba-9170-21f6e5572570.XML
[14:55:16] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/AS_NORMATIVE_DOCS_TYPES_20231026_ec4bc25a-31a1-4ec8-805b-eacabe781aab.XML
[14:55:16] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/AS_OBJECT_LEVELS_20231026_10c61fea-632f-4017-91ab-200ed516b142.XML
[14:55:16] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/AS_OPERATION_TYPES_20231026_bf3006f0-f7c5-4e29-aadb-3671d5b3fe59.XML
[14:55:16] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/AS_PARAM_TYPES_20231026_88d99e07-ce03-41e4-9288-f1ebfaa80c26.XML
[14:55:17] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/AS_ROOM_TYPES_20231026_a8f37b4d-bd22-4f73-a8a8-f489efdc4f2b.XML
[14:55:17] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/01/AS_ADDR_OBJ_20231026_8f6f03d3-5d66-4089-9a0c-17dae2e61210.XML
[14:55:21] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/01/AS_ADDR_OBJ_DIVISION_20231026_c7b37f78-fbe7-45c6-8c3c-8569b4a4bb96.XML
[14:55:21] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/01/AS_ADDR_OBJ_PARAMS_20231026_7235f092-e664-47e6-8558-73d0d29591db.XML
[14:56:00] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/01/AS_ADM_HIERARCHY_20231026_19413821-8a61-49dd-bb1d-9517ff8ff13c.XML
[14:58:46] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/01/AS_APARTMENTS_20231026_1cca59be-ed09-406c-99cc-9055e9e215bd.XML
[14:59:31] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/01/AS_APARTMENTS_PARAMS_20231026_fe757930-1a5f-47e2-b047-8c570e74cae4.XML
[15:01:00] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/01/AS_CARPLACES_20231026_e942c2bb-0fd0-4e20-8714-d5bdb8da5ad3.XML
[15:01:00] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/01/AS_CARPLACES_PARAMS_20231026_525d3d4d-652a-490b-b6b7-647e16a37f11.XML
[15:01:01] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/01/AS_CHANGE_HISTORY_20231026_9ab53d26-f9af-4713-a551-f35669091c9b.XML
[15:07:09] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/01/AS_HOUSES_20231026_4fb8893b-3b6e-409b-b719-58dda0b3e9dc.XML
[15:10:26] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/01/AS_HOUSES_PARAMS_20231026_6315849a-fe14-460a-9fc9-6868707bccea.XML
[15:24:26] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/01/AS_MUN_HIERARCHY_20231026_0773a20b-025c-4baa-8bc1-b8819e5d3b33.XML
[15:26:33] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/01/AS_NORMATIVE_DOCS_20231026_49f66abd-68c5-4b8d-9d02-c219562f2e9f.XML
[15:26:53] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/01/AS_REESTR_OBJECTS_20231026_b391436a-a026-4c65-bd19-4ffebc6de3e7.XML
[15:28:19] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/01/AS_ROOMS_20231026_cb613eab-767d-460e-870a-695f01cf9493.XML
[15:28:19] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/01/AS_ROOMS_PARAMS_20231026_fee7facc-8423-4ba9-9065-0a53d53deb3a.XML
[15:28:19] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/01/AS_STEADS_20231026_63a27047-ffe2-4844-8424-2b979a72c749.XML
[15:28:39] ... /media/rootadmin/Transcend/FIAS_GAR_2023_10_26/01/AS_STEADS_PARAMS_20231026_efa2bc05-7dc2-404e-82a3-89a6ae9fd11b.XML
[15:31:49] ...  -- Data finish
[15:31:49] ...  -- Set LOGGED
[15:32:23] ... Stop process
--------------------------------------------------------
2023-11-02 17:00:33 ./
127.0.0.1 5434 unsi_test_01 postgres stage_3_01.csv 2023-10-26
--------------------------------------------------------
[17:00:33] ... Start process, Подготовка
[17:00:33] ... -- Проверка региона
[17:00:33] ...  -- Set LOGGED
[17:00:34] ...  -- Установка индексного покрытия в схеме в схеме GAR_FIAS
[17:00:40] ...  -- Очистка данных во временной схеме
[17:00:43] ... -- Заполнение таблицы с дефектными данными
[17:00:43] ...  -- Обработка таблицы с дефектными данными
[17:00:43] ...  -- Ещё раз проверка дублей
[17:00:44] ... Stop process
--------------------------------------------------------
2023-11-03 11:43:38 ./
127.0.0.1 5434 unsi_test_01 postgres stage_31_01.csv 2023-10-26
--------------------------------------------------------
[11:43:38] ... Start process, Подготовка
[11:43:38] ... -- Проверка региона
[11:43:38] ...  Установка gar_tmp.obj_seq/gar_tmp.obj_hist_seq
[11:43:39] ...  -- Агрегация параметров
[11:43:50] ...  -- Агрегация адресных пространств
[11:43:53] ...  -- Агрегация домовладений
[11:43:59] ...  -- Заполнение рабочей таблицы "gar_tmp.xxx_obj_fias"
[11:44:04] ... Stop process
--------------------------------------------------------
