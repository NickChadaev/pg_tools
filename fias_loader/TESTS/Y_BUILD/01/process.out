 save_gar_version 
------------------
               79
(1 строка)

CALL
CALL
 set_gar_dt_create 
-------------------
                79
(1 строка)

CALL
 id_area | id_country | nm_area | nm_area_full | id_area_type | id_area_parent | kd_timezone | pr_detailed | kd_oktmo |             nm_fias_guid             | dt_data_del | id_data_etalon |  kd_okato   | nm_zipcode |  kd_kladr   | vl_addr_latitude | vl_addr_longitude 
---------+------------+---------+--------------+--------------+----------------+-------------+-------------+----------+--------------------------------------+-------------+----------------+-------------+------------+-------------+------------------+-------------------
       2 |        185 | Адыгея  | Адыгея Респ  |           41 |                |           2 |           0 | 79000000 | d8327a56-80de-4df2-815c-4f6ab1224c50 |             |                | 79000000000 | 385000     | 01000000000 |                  |                  
(1 строка)

CALL
CALL
CALL
 f_addr_area_set_data 
----------------------
                    8
(1 строка)

            fias_guid_new             |            fias_guid_old             | nm_addr_obj | addr_obj_type_id | chld_qty_tot | obj_level 
--------------------------------------+--------------------------------------+-------------+------------------+--------------+-----------
 9fd308f2-0ae8-4e9f-89cd-8dc833cdf415 | 5c8405c9-b73a-44e1-b82c-acf8211873be | ВЕСЕЛЫЙ     |              112 |           10 |         6
 96fe8f29-c1e6-43d7-a1d7-f327bef75c63 | 01c0f03e-bd21-4ce8-86a2-9ae3259143fe | ГРОЗНЫЙ     |              112 |            2 |         6
 29fe3efe-8f3a-4a61-bf98-481e3acca90e | 145949fd-0aa4-4bad-8c83-3b6701972716 | КОЧКИН      |              112 |            2 |         6
 cf29e918-e413-4f6c-8391-148176db582d | 294468ce-fb2c-4422-9765-0b8b0d6026f6 | ПОБЕДА      |               90 |            3 |         6
(4 строки)

 id_addr_obj | id_addr_parent |              fias_guid               |           parent_fias_guid           |       nm_addr_obj        | addr_obj_type_id | addr_obj_type | obj_level |          level_name          | region_code | area_code | city_code | place_code | plan_code | street_code | change_id | prev_id | oper_type_id | oper_type_name | start_date |  end_date  | id_lead |       tree_d        | level_d 
-------------+----------------+--------------------------------------+--------------------------------------+--------------------------+------------------+---------------+-----------+------------------------------+-------------+-----------+-----------+------------+-----------+-------------+-----------+---------+--------------+----------------+------------+------------+---------+---------------------+---------
        2639 |           2416 | 31295a7b-1277-4b82-a40c-6be7d0cbd850 | 9fd308f2-0ae8-4e9f-89cd-8dc833cdf415 | А/Д Подъезд к х. Веселый |              127 | дор           |         8 | Элемент улично-дорожной сети | 1           | 4         | 0         | 4          | 0         | 10          |      7517 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |    3662 | {11,2157,2416,2639} |       4
        3662 |           2416 | 096cf142-7d31-428e-8b43-095bfa1c1670 | 9fd308f2-0ae8-4e9f-89cd-8dc833cdf415 | А/Д Подъезд к х. Веселый |              127 | дор           |         8 | Элемент улично-дорожной сети | 1           | 4         | 0         | 5          | 0         | 12          |     11198 |       0 |            1 | Инициация      | 1900-01-01 | 2079-06-06 |    3662 | {11,2157,2416,3662} |       4
(2 строки)

 id_area | id_country | nm_area | nm_area_full | id_area_type | id_area_parent | kd_timezone | pr_detailed | kd_oktmo |             nm_fias_guid             | dt_data_del | id_data_etalon |  kd_okato   | nm_zipcode |  kd_kladr   | vl_addr_latitude | vl_addr_longitude 
---------+------------+---------+--------------+--------------+----------------+-------------+-------------+----------+--------------------------------------+-------------+----------------+-------------+------------+-------------+------------------+-------------------
       2 |        185 | Адыгея  | Адыгея Респ  |           41 |                |           2 |           0 | 79000000 | d8327a56-80de-4df2-815c-4f6ab1224c50 |             |                | 79000000000 | 385000     | 01000000000 |                  |                  
(1 строка)

 f_xxx_obj_seq_crt 
-------------------
         200008952
       10200008952
(2 строки)

 f_set_params_value 
--------------------
             309150
(1 строка)

 f_xxx_adr_area_set_data 
-------------------------
                    6160
(1 строка)

 f_xxx_adr_house_set_data 
--------------------------
                   150783
(1 строка)

 f_xxx_obj_fias_set_data 
-------------------------
                     434
                    5726
                  150783
(3 строки)

